sdk/core/core-paging/src/getPagedAsyncIterator.ts

```diff
@@ -67,9 +67,15 @@ async function* getPageAsyncIterator<TPage, TLink, TPageSettings>(
 ): AsyncIterableIterator<TPage> {
   const { pageLink, maxPageSize } = options;
   let response = await pagedResult.getPage(pageLink ?? pagedResult.firstPageLink, maxPageSize);
+  if (!response) {
+    return;
+  }
```

Comment:
could it be the case that response is a falsey value that doesn't necessarily indicate absence of a page?, e.g. an empty string or false.

Reply:
oh please ignore my comment, `getPage` returns either an object or an undefined so there is no room for other falsey values.